results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+
+## What was tested
+
+**Operating systems:** iOS, Mac OS Monterey, Windows 10, Windows 11
+
+**Browsers:** Chrome, Firefox, Opera GX, Safari
+
+**Assistive tech:** Built-in large cursor and pointer operating system settings, customized operating system shortcuts, Dragon Professional (speech recognition software), in-browser contrast adjustments, in-browser focus styling adjustments, Mac OS Zoom (built-in screen magnifier), mobile device, split keyboard, [sticky keys](https://en.wikipedia.org/wiki/Sticky_keys)
+
+**Interface:** Participant browser and [preset environment in The Littlest JupyterHub](https://github.com/Quansight-Labs/JupyterLab-user-testing#jupyterlab-user-testing-environment) running JupyterLab 3.6.1. 
+
+**Sample size:** 7 participants
+
+**Method:** Combination of qualitative [usability testing](https://www.nngroup.com/articles/usability-testing-101/) and [user interviews](https://www.nngroup.com/articles/user-interviews/)
+
+**Team:** [Isabela Presedo-Floyd](https://github.com/isabela-pf), [Tania Allard](https://github.com/trallard/), [Stephannie Jimenez Gacha](https://github.com/steff456), and [Gabriel Fouasnon](https://github.com/gabalafou/) via [Quansight Labs](https://labs.quansight.org/)
+
+## Limits
+
+While this study provides insights into how JupyterLab may function for users of assistive technology, it is important to explicitly note limits and gaps in this specific study.
+
+- **We worked with a small sample size of seven participants.** These participants joined as a part of an open call, so they are self-selected. Facilitators did select from within the larger group to make sure participants had used Jupyter tools before and were in a region where we could send payment.
+- **None of the participants used a screen reader.** Screen readers are a prominent group of assistive tech and critical to test with, so an ideal sample would absolutely include them. Of the assistive tech used by participants in this study, none of them present notebook content back to their users in a strictly non-visual way. For example, a screen magnifier user does interact with the notebook very differently than someone working without magnification. But both screen magnifier users and default magnification users are using their vision to read what is on the screen; they are able to directly interact with visual information. A screen reader user would have that content interpreted by the screen reader and read to them; they are interacting with an interpretation of visual information which is often where major issues arise.
+- **This study is not an attempt to determine JupyterLab’s compliance with any single accessibility standard.** These results are not an accessibility statement for JupyterLab, JupyterHub, or any related Jupyter projects.
```

Comment:
```suggestion
- **This study is not an attempt to determine JupyterLab’s conformance with any accessibility standard.** These results are not an accessibility statement for JupyterLab, JupyterHub, or any related Jupyter projects.
```
I'd like to suggest this rewording instead, I have come to realise this is a common confusion, and terms are often used interchangeably. So in the interest of all of us, using a common vocabulary:

- conformance: refers to expectations, standards, and policies that a particular company or product abides by (defined by the organisation itself or an external entity).
- compliance: requirements that an organisation **must** meet and are established by an external authority or governing jurisdiction. 

While compliance is a more formal and legal requirement, conformance is voluntary.

Example:
For JupyterLab to be legally compliant with the UK Government accessibility regulations, it would need to be WCAG conformant.

Reply:
My mistake.